packages/bpk-tokens/src/web/base/flares.json

```diff
@@ -0,0 +1,18 @@
+{
+  "imports": ["./aliases.json"],
+  "global": {
+    "type": "size",
+    "category": "flare"
+  },
+  "props": {
+    "FLARE_HEIGHT_DESKTOP": {
+      "value": "{!SPACING_BASE}"
+    },
+    "FLARE_HEIGHT_MOBILE": {
+      "value": "{!SPACING_MD}"
+    },
+    "FLARE_CORNER_RADIUS": {
+      "value": "{!BORDER_RADIUS_PILL_LG}"
```

Comment:
This value is about to be deprecated, be aware this will need to change as it will become `4px`

Reply:
Ignore me this is invalid and got mixed up with BUTTON_BORDER_RADIUS. 🤦‍♂ 

![](https://media2.giphy.com/media/xT1XGWbE0XiBDX2T8Q/giphy.gif)